docs/conf.py

```diff
@@ -87,7 +88,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
I've become a fan of the [furo](https://github.com/pradyunsg/furo) theme. Two words... dark mode.